src/C-interface/bml_utilities.c

```diff
@@ -529,3 +529,18 @@ bml_write_bml_matrix(
             break;
     }
 }
+
+// compute largest int "i" such that i*i <= x
```

Comment:
Wouldn't
```
(int) floor(sqrt(x)))
```
Be a little simpler? Or are you concerned about performance and this implementation is faster?

Reply:
Performance is not an issue since it is run only during setup. I just liked the idea of having an "int" specific function

Reply:
The type casting is a bit annoying, I agree. It might be more transparent when reading the code though. I am ok with either solution though. I was just interested in understanding why you had chosen this way of implementing it. Thanks!